Running cluster for deep fits
120014 120015 normal
Running object 120014
Not enough g observations!
WARNING: Could not memory map array with mode='readonly', falling back to mode='denywrite', which means that the array will be read-only [astropy.io.fits.file]
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  39
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  75
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  117
Counts: {'r': 36, 'i': 69, 'z': 113}
Mu: {'g': 0.0, 'r': 0.1377756, 'i': 0.23812535, 'z': 0.3936921}
No handles with labels found to put in legend.
Variable??
0
0
Non-Variable source in g
Variable??
0.02168679004535079
1.6441947650794644
Non-Variable source in r
Variable??
0.03144590304348349
1.3117423266351533
Non-Variable source in i
Variable??
0.06564059338262458
2.771995333134743
Non-Variable source in z
Variable:    0/4
Var Bands:   
Slope:       [1, 1, 0.24214576183945002, 0.7254494582436102]
Slope Error: [0.0, 0, 0.09635421920176336, 0.19369035019156997]
intercepts:  [0, 0, -0.041614782142663884, -0.10217065409864334]
Int Error:   [0.0, 0, 0.04498369006147536, 0.050763378796507254]
StDev:       [0.2417747, 0.049376737, 0.05759015, 0.10562196]
Not variable, closing plots & going to next row
Successfully finished!
Rows 120014 to 120015 ran in 2.5467 seconds
